warehouse/oidc/models/__init__.py

```diff
@@ -27,4 +31,5 @@
     "GitHubPublisher",
     "GooglePublisher",
     "ActiveStatePublisher",
+    "OIDCPublisherMixin",
```

Comment:
Is this necessary? `OIDCPublisherMixin` is only used in `warehouse.oidc.models._core`, why do we need this?

Reply:
I can't recall off the top of my head.  I'll try removing it and running linters and tests to see if it can be removed.

Reply:
Everything seemed happy after removing this so removed and pushed.